meta-facebook/yv4-ff/src/platform/plat_isr.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef PLAT_ISR_H
+#define PLAT_ISR_H
+
+#include <stdint.h>
+#include "ipmi.h"
+
+// BIC checks for CXL readiness every three seconds, with a maximum of 10 attempts
+#define CXL_READY_RETRY_TIMES 10
+#define CXL_READY_INTERVAL_SECONDS 3
+#define CXL_HEART_BEAT_LABEL "HB0"
+
+void ISR_MB_DC_STAGUS_CHAGNE();
```

Comment:
typo - void ISR_MB_DC_STATUS_CHANGE();

Reply:
The function name is revised. Please check. 
Thanks.